database/src/main/postgres/runs/V1.8.3__get_partitioning_checkpoints.sql

```diff
@@ -28,53 +28,64 @@ CREATE OR REPLACE FUNCTION runs.get_partitioning_checkpoints(
     OUT measured_columns           TEXT[],
     OUT measurement_value          JSONB,
     OUT checkpoint_start_time      TIMESTAMP WITH TIME ZONE,
-    OUT checkpoint_end_time        TIMESTAMP WITH TIME ZONE
+    OUT checkpoint_end_time        TIMESTAMP WITH TIME ZONE,
+    OUT has_more                   BOOLEAN
 )
-    RETURNS SETOF record AS
-$$
-    -------------------------------------------------------------------------------
+RETURNS SETOF record AS
+-------------------------------------------------------------------------------
 --
--- Function: runs.get_partitioning_checkpoints(JSONB, INT, TEXT)
---      Retrieves all checkpoints (measures and their measurement details) related to a
+-- Function: runs.get_partitioning_checkpoints(4)
+--      Retrieves checkpoints (measures and their measurement details) related to a
 --      given partitioning (and checkpoint name, if specified).
 --
 -- Parameters:
 --      i_partitioning          - partitioning of requested checkpoints
---      i_limit                 - (optional) maximum number of checkpoint's measurements to return
---                                if 0 specified, all data will be returned, i.e. no limit will be applied
+--      i_limit                 - (optional) maximum number of checkpoints to return
+--      i_offset                - (optional) offset of the first checkpoint to return
```

Comment:
I would consider mentioning in this code doc - maybe here for `i_offset` even - based on which the checkpoints are being returned (the sorting logic)

Reply:
ok added note about these params in the docs